src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    public static final char GIVEN_CHARACTER = 'w';
+    public static final String WHITE_SPACE = " ";
+    public static final String EMPTY = "";
```

Comment:
if we won't use variables outside the class let's make them private to follow encapsualtion principle

Reply:
done.